docs/source/create_channel/create_channel_test_net.md

```diff
@@ -0,0 +1,444 @@
+```
+osnadmin channel join --channelID channel1 --config-block ./channel-artifacts/channel1.block -o localhost:7053 --ca-file "$ORDERER_CA" --client-cert "$ORDERER_ADMIN_TLS_SIGN_CERT" --client-key "$ORDERER_ADMIN_TLS_PRIVATE_KEY"
+```
+
+- **`--channelID`**: Specify the name of the application channel that you provided when you created the channel genesis block.
+- **`--config-block`**: Specify the location of the channel genesis block that you created with the `configtxgen` command, or the latest config block.
+- **`-o`**: Specify the hostname and port of the orderer admin endpoint. For the test network ordering node this is set to `localhost:7053`.
+
+In addition, because the `osnadmin channel` commands communicate with the ordering node using mutual TLS, you need to provide the following certificates:
+- **`--ca-file`**: Specify the location and file name of the orderer organization TLS CA root certificate.
+- **`--client-cert`**: Specify the location and file name of admin client signed certificate from the TLS CA.
+- **`--client-key`**: Specify the location and file name of admin client private key from the TLS CA.
+
+When successful, the output of the command contains the following:
+```
+Status: 201
+{
+	"name": "channel1",
+	"url": "/participation/v1/channels/channel1",
+	"consensusRelation": "consenter",
+	"status": "active",
+	"height": 1
+}
+```
+
+The channel is active and ready for peers to join.
+
+### Consenter vs. Follower
+
+Notice the ordering node was joined to the channel with a `consensusRelation: "consenter"`. If you ran the command against an ordering node that is not included in the list of `Consenters:` in the `configtx.yaml` file (or the channel configuration consenter set), it is added to the channel as a `follower`. This is important because anytime a new orderer joins the cluster, you need to ensure that you do not lose quorum while the ledger on the new orderer catches up. Adding a second orderer to a single-node cluster changes the majority from one to two. Therefore, during the catch-up process and until the new orderer starts executing consensus, the cluster is in a state of reduced fault tolerance. To account for this situation and avoid any downtime, the `osnadmin channel list` command can be used to ascertain the consensusRelation status of an orderer, which can be either `consenter` or `follower`. When a node joins as a `follower`, the channel ledger is replicated from other orderers, but as it does not take part in consensus, channel operations are not impacted. Because replicating a long chain of blocks can take a long time, joining as a `follower` is most useful for channels with high block heights because it allows you to confirm that an orderer is able to replicate blocks before having it count towards quorum. Therefore, to join an orderer to a channel as a `follower`, you should not include the node in the channel configuration consenter set.
```

Comment:
Isn't this information elsewhere in the docs? Couldn't we link to it instead of describe the whole thing again here? Would nice to keep the test network tutorial short and sweet for newbies. I think it is worth mentioning in a sentence or two, and then link to the details.

Reply:
This comment reinforces the need for another conceptual topic or ops guide content. I left 2 sentences in and then linked to where we currently explain more on this.